<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowStart</class>
 <widget class="QMainWindow" name="MainWindowStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>880</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>csNova - Start</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>36</number>
      </property>
      <widget class="QWidget" name="widgetLeft" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="1">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>400</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>(c)2025 - csNova</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetCenter" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="welcomeTxt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Lato</family>
            <pointsize>20</pointsize>
            <bold>true</bold>
            <stylestrategy>PreferAntialias</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Willkommen bei CSNova</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelWelcome">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Lato</family>
            <pointsize>12</pointsize>
            <bold>false</bold>
            <stylestrategy>PreferAntialias</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vielen Dank, dass du dich für &lt;span style=&quot; font-weight:700;&quot;&gt;csNova&lt;/span&gt; entschieden hast. &lt;/p&gt;&lt;p&gt;&amp;quot;Codices Scriptoria Nova&amp;quot; - &lt;span style=&quot; font-style:italic;&quot;&gt;Bücher die in neuen Schreibwerkwerstätten entstehen &lt;/span&gt;- in &lt;span style=&quot; font-weight:700;&quot;&gt;csNova&lt;/span&gt;.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;csNova&lt;/span&gt; ist ein Open-Source-Projekt, dass dich dabei unterstützt, deine kreativen Schreibprojekte zu organisieren und zu verwalten.&lt;/p&gt;&lt;p&gt;Dabei passt sich das Programm deinen Arbeitsstil an - einfach nach dem Motto:&lt;br/&gt;&amp;quot;Nicht du musst lernen, wie das Programm funktioniert - das Programm soll &lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;verstehen&lt;/span&gt;, wie du arbeiten willst&amp;quot;&lt;/p&gt;&lt;p&gt;Bevor du loslegt, kannst du erst einmal einige Grundeinstellungen vornehmen:&lt;br/&gt;&lt;/p&gt;&lt;p&gt;- csNova hat ermittelt, in welcher Sprache dein Rechner grundsätzlich arbeitet. Diese Voreinstellung kannst du jetzt ändern&lt;/p&gt;&lt;p&gt;- Du hast verschiedene Designs zur Auswahl. Wähle die Optik, die dir gefällt&lt;/p&gt;&lt;p&gt;- Klicke anschließend auf &lt;span style=&quot; font-weight:700;&quot;&gt;Speichern&lt;/span&gt; - auch wenn du nichts ändern möchtest - und dann kannst du direkt loslegen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::RichText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelLanguages">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
               <stylestrategy>PreferAntialias</stylestrategy>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Sprachen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBoxLanguage">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Deutsch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Englisch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Französisch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spanisch</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelTheme">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
               <stylestrategy>PreferAntialias</stylestrategy>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>Design</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QComboBox" name="comboBoxTheme">
             <property name="currentIndex">
              <number>7</number>
             </property>
             <item>
              <property name="text">
               <string>Antik (hell)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Antik (neutral)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Antik (dunkel)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Oldschool (hell)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Oldschool (neutral)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Oldschool (dunkel)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modern (hell)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modern (neutral)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modern (dunkel)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vintage (hell)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vintage (neutral)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vintage (dunkel)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Futuristisch (hell)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Futuristisch (neutral)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Futuristisch (dunkel)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Minimalistisch (hell)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Minimalistisch (neutral)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Minimalistisch (dunkel)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Lato</family>
            <pointsize>12</pointsize>
            <bold>false</bold>
            <stylestrategy>PreferAntialias</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diese und weitere Einstellungen kannst du später und jederzeit anpassen. Klicke dazu einfach auf die Schaltläche &lt;span style=&quot; font-weight:700;&quot;&gt;EINSTELLUNGEN&lt;/span&gt;. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Die aktuellen Einstellungen von Sprache und Design übernehmen</string>
          </property>
          <property name="text">
           <string>Speichern</string>
          </property>
          <property name="icon">
           <iconset theme="document-save"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetRight" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="projectBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Erstellen, bearbeiten und verwalten von Projekten</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Projekte</string>
          </property>
          <property name="icon">
           <iconset theme="video-x-generic"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="characterBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Erstellen, bearbeiten und verwalten von Charakteren</string>
          </property>
          <property name="text">
           <string>Charaktere</string>
          </property>
          <property name="icon">
           <iconset theme="face-cool"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="objectsBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Erstellen, bearbeiten und verwalten von Objekten</string>
          </property>
          <property name="text">
           <string>Objekte</string>
          </property>
          <property name="icon">
           <iconset theme="applications-accessories"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="locationBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Erstellen, bearbeiten und verwalten von Orten</string>
          </property>
          <property name="text">
           <string>Handlungsorte</string>
          </property>
          <property name="icon">
           <iconset theme="emblem-photos"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="storylineBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Erstellen, bearbeiten und verwalten von Storylines</string>
          </property>
          <property name="text">
           <string>Handlungsstränge</string>
          </property>
          <property name="icon">
           <iconset theme="emblem-documents"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editorBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Erstellen, bearbeiten und verwalten von Kapiteln und Szenen</string>
          </property>
          <property name="text">
           <string>Editor</string>
          </property>
          <property name="icon">
           <iconset theme="accessories-text-editor"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="preferencesBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Einstellungen anpassen: Sprache, Design, Vorlagen uvm.</string>
          </property>
          <property name="text">
           <string>Einstellungen</string>
          </property>
          <property name="icon">
           <iconset theme="applications-utilities"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Tutorials, Hilfen, Tipps, Kontakt und allgemeine Information zu csNova</string>
          </property>
          <property name="text">
           <string>Hilfen + Tipps</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-question"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>120</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exitBnt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Das Programm csNova beenden</string>
          </property>
          <property name="text">
           <string>csNova beenden</string>
          </property>
          <property name="icon">
           <iconset theme="application-exit"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
